launcher/commands/install.py

```diff
@@ -110,7 +110,7 @@ def _install_mod_organizer(self, version: str) -> None:
 
     def run(self, args) -> None:
         if not args.cache_path:
```

Comment:
Should be removed since extraction in tmp directory will always happen

See:
https://github.com/Mord3rca/gamma-launcher/blob/master/launcher/mods/downloader/github/git.py#L37
https://github.com/Mord3rca/gamma-launcher/blob/master/launcher/mods/downloader/github/legacy.py#L35